### PR TITLE
Convert MPI_Gather.3in - MPI_Get_version.3in to md

### PR DESCRIPTION
**Pull request by Fangcong-Yin**
10 files are converted in total.
MPI_Get_elements_x.3in was skipped because it is empty.
Signed-off-by: Fangcong Yin (fyin2@nd.edu)

### CODE REVIEW

File: ompi/mpi/man/man3/MPI_Gather.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+Each process (root process included) sends the contents of its send
+buffer to the root process. The root process receives the messages and
+stores them in rank order. The outcome is as if each of the n processes
+in the group (including the root process) had executed a call to
+
+    MPI_Send(sendbuf, sendcount, sendtype, root, ...)
```
[the remark this answers is not in the document]
Comment (Fangcong-Yin) [reply ompi/mpi/man/man3/MPI_Gather.md:92]: Just fixed!

File: ompi/mpi/man/man3/MPI_Gather.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+and the root had executed n calls to
+
+    MPI_Recv(recfbuf + i * recvcount * extent(recvtype), recvcount, recvtype, i, ...)
+
+where extent(recvtype) is the type extent obtained from a call to
+`MPI_Type_extent()`.
+
+An alternative description is that the n messages sent by the processes
+in the group are concatenated in rank order, and the resulting message
+is received by the root as if by a call to MPI_RECV(recvbuf, recvcount
```
[the remark this answers is not in the document]
Comment (Fangcong-Yin) [reply ompi/mpi/man/man3/MPI_Gather.md]: Juste fixed!

File: ompi/mpi/man/man3/MPI_Gatherv.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+Before the error value is returned, the current MPI error handler is
+called. By default, this error handler aborts the MPI job, except for
+I/O function errors. The error handler may be changed with
+`MPI_Comm_set_errhandler`; the predefined error handler `MPI_ERRORS_RETURN`
+may be used to cause error values to be returned. Note that MPI does not
+guarantee that an MPI program can continue past an error.
+
+# See Also
+
+[``(3)](.html)
```
[the remark this answers is not in the document]
Comment (Fangcong-Yin) [reply ompi/mpi/man/man3/MPI_Gatherv.md]: I'm sorry that this is a redundant line that I forgot to delete. Just deleted!

File: ompi/mpi/man/man3/MPI_Gather.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsquyres) [reply ompi/mpi/man/man3/MPI_Gather.md:92]: Sorry to be picky -- can you remove the indenting, too?

[the remark this answers is not in the document]
Comment (Fangcong-Yin) [reply ompi/mpi/man/man3/MPI_Gather.md]: No problem!

File: ompi/mpi/man/man3/MPI_Gather.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+MPI_Recv(recfbuf + i * recvcount * extent(recvtype), recvcount, recvtype, i, ...)
+```
+
+where extent(recvtype) is the type extent obtained from a call to
+`MPI_Type_extent()`.
+
+An alternative description is that the n messages sent by the processes
+in the group are concatenated in rank order, and the resulting message
+is received by the root as if by a call to `MPI_RECV(recvbuf, recvcount
+* n, recvtype, ... )`.
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Gather.md:107]: Something went wrong here -- when you look at the rendered version, it doesn't look right.  Is the `*n` supposed to be there?  That might be throwing off the Markdown rendering (i.e., the Markdown renderer might be getting confused by a as the first character and choosing to render it as a bullet).

File: ompi/mpi/man/man3/MPI_Gatherv.md [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+message to the root,
+
+```c
+MPI_Send(sendbuf, sendcount, sendtype, root, ...)
+```
+
+and the root executes n receives,
+
+```c
+MPI_Recv(recvbuf + disp[i] * extent(recvtype), \
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Gatherv.md:109]:
```suggestion
MPI_Recv(recvbuf + disp[i] * extent(recvtype),
```

File: ompi/mpi/man/man3/MPI_Gather.md (hunk 4 above)
Comment (Fangcong-Yin) [reply ompi/mpi/man/man3/MPI_Gather.md]: Got it! I moved the * to line 106 right after recvcount and keep "n, recvtype, ...) in line 107.

File: ompi/mpi/man/man3/MPI_Gatherv.md (hunk 5 above)
Comment (Fangcong-Yin) [reply ompi/mpi/man/man3/MPI_Gatherv.md]: Sorry for the additional \! Just deleted it.

**Review by jsquyres — APPROVED** [review]

**Comment by jsquyres** [comment]
ok to test

**Pull request merged**